### PR TITLE
sqlserver jdbc url should be supported by container provider

### PR DESCRIPTION
**Pull request by neetkee**
Solution to https://github.com/testcontainers/testcontainers-java/issues/988

### CODE REVIEW

File: modules/mssqlserver/src/test/java/org/testcontainers/containers/SqlServerJDBCDriverTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.apache.commons.dbutils.QueryRunner;
+import org.apache.commons.dbutils.ResultSetHandler;
+import org.junit.Test;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+
+public class SqlServerJDBCDriverTest {
+
+    @Test
+    public void testMsSqlServerConnection() throws Exception {
```
[the remark this answers is not in the document]
Comment (neetkee) [reply modules/mssqlserver/src/test/java/org/testcontainers/containers/SqlServerJDBCDriverTest.java]: Not sure I get it, the test contains assertEquals.

**Comment by neetkee** [comment]
It's possible to use JDBC url with `mssqlserver`, [mentioned it here](https://github.com/neetkee/tc-mssql-naming/blob/master/src/test/resources/application.yml#L6)

**Comment by kiview** [comment]
@StefanHufschmidt Can you give it a look please?

**Comment by StefanHufschmidt** [comment]
Btw do we automatically accept the license agreement on this ms sql server image in our tests?
Might that cause some problems for us?

**Comment by kiview** [comment]
@StefanHufschmidt You mean legal problems?
We accept it in the `jdbc-test` module.

**Comment by kiview** [comment]
@neetkee This will be a breaking API change, correct?
Would it make sense, to support both names?

**Comment by neetkee** [comment]
@kiview Yes, it will be the breaking change for those, who use this image as temporary database container. But I'm not sure if we need to keep `mssqlserver `jdbc url working, documentation says that to work with it, it's only needed to `Insert tc: after jdbc:`

**Comment by bsideup** [comment]
Pinging @kiview

**Comment by kiview** [comment]
@neetkee Whoops sorry, everything looks good.
Since this is kind of API breaking, we should mention it in the release notes, but it's the right thing to do.

Thanks for your contribution!

**Pull request merged**